core-tests/src/test/java/org/mule/MessagePropertiesContextTestCase.java

```diff
@@ -121,36 +94,18 @@ public void testCopyConstructor() throws Exception
         assertSame(mpc.getProperty("ABC", PropertyScope.OUTBOUND), "abc");
     }
 
-    /*@Test
-    public void testInboundScopeIsImmutable() throws Exception
```

Comment:
From what i remember this immutability was never implemented, but seems the test method was commented out instead of removed.